.gitignore

```diff
@@ -1,7 +1,9 @@
-__pycache__/
-*.pyc
-/.cache_metadata.json
-.coverage
-/.venv/
-/data/
-/cache/
+__pycache__/
+*.pyc
+/.cache_metadata.json
+.coverage
+/.venv/
+/data/
+/cache/
+/.vscode/
+/.venv-wsl/

```

Comment:
I don't see a reason to have this? Added by accident?

Reply:
wsl is the Windows Subsystem for Linux. Truewiki has issues with windows filepath separators so I run it on my windows machine inside WSL. I'll move it to the global gitignore (which I just learned was a thing)